src/ray/common/asio/periodical_runner.cc

```diff
@@ -104,7 +103,7 @@ void PeriodicalRunner::DoRunFnPeriodicallyInstrumented(
   // event loop.
   auto stats_handle = io_service_.stats().RecordStart(name, period.total_nanoseconds());
   timer->async_wait([this,
-                     fn = std::move(fn),
```

Comment:
why removing move?

Reply:
because that fn move did not work, because fn is a const& and move is no-op.